src/Castle.Core.Tests.WeakNamed/DynamicProxy.Tests/BasicClassProxyTestCase.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright 2004-2018 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.DynamicProxy.Tests
+{
+	using System;
+	using System.Collections.Generic;
+	using System.Reflection;
+
+	using Castle.DynamicProxy.Generators.Emitters;
+
+	using NUnit.Framework;
+
+	[TestFixture]
+	public class BasicClassProxyTestCase : BasePEVerifyTestCase
+	{
+		[Test]
+		public void ProxyForBaseTypeFromUnsignedAssembly()
+		{
+			Assert.False(TestAssemblySigned());
```

Comment:
Could we move this into a test fixture set up and add a reason message parameter, or just get rid of the assertion completely as it is no longer really needed since this whole project has this requirement now?

Reply:
Done. Let's just drop these assertions, then. Like you said, the test project itself already guarantees weak-named-ness. Even if a future maintainer were tempted to move these tests back to the original fixture (now that he explicit assertion is no longer there to prevent this), the tests would still start failing, so that should be enough of a safeguard.